### PR TITLE
bug-fixing

### PR DESCRIPTION
**Pull request by roman454647**

### CODE REVIEW

File: client/src/components/ChangePassword/index.jsx [hunk 1]
```diff
@@ -59,43 +59,42 @@ class ChangePassword extends Component {
...
-              />
-              <span className='help-block'>{validations.newPassword.message}</span>
-              <span
-                className='password-show'
-                role='presentation'
-                onClick={this.handleShowHide}
-              ><img src={eyeicon} alt='show/hide icon' /></span>
-            </div>
+        <div className='change-password-form'>
+          <div className={validations.oldPassword.isInvalid ? 'has-error' : 'undefined'}>
```
Comment (anna-kuzii) [inline client/src/components/ChangePassword/index.jsx]: Class undefined?

File: client/src/components/UpdateProfile/index.jsx [hunk 2]
```diff
@@ -21,7 +21,7 @@ export const UpdateProfile = () =>(
           name='email'
         />
       </div>
-      <button type='button' className='update-profile'>Update Profile</button>
+      <button type='button' className='update-profile'>UPDATE PROFILE</button>
```
Comment (anna-kuzii) [inline client/src/components/UpdateProfile/index.jsx]: Lol. Do this using css

File: client/src/components/UserProfile/index.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  BrowserRouter as Router,
+  Switch,
+} from 'react-router-dom';
+import { ChangePassword } from '../ChangePassword';
+import { UpdateProfile } from '../UpdateProfile';
+import { Header } from '../Header';
+import { Menu } from '../Menu';
+import menuData from '../../assets/data/menuItems.json';
+
+const UserProfileComponent =()=> (
```
Comment (anna-kuzii) [inline client/src/components/UserProfile/index.jsx]: U missed spaces

File: client/src/components/UpdateProfile/index.jsx (hunk 2 above)
Comment (roman454647) [reply client/src/components/UpdateProfile/index.jsx]: changed

File: client/src/components/UserProfile/index.jsx (hunk 3 above)
Comment (roman454647) [reply client/src/components/UserProfile/index.jsx]: changed

File: client/src/components/ChangePassword/index.jsx (hunk 1 above)
Comment (roman454647) [reply client/src/components/ChangePassword/index.jsx]: changed

**Pull request merged**